src/lib/objects.js

```diff
@@ -0,0 +1,67 @@
+const createPerson = (name, age) => {
+  const person = {
+    name: name,
+    age: age
```

Comment:
Since you are using this same shorthand in the `createTalkingPerson` function below, it makes sense to use it here also for consistency
```
    name,
    age,
```